docs/maps/maps-getting-started.asciidoc

```diff
@@ -34,8 +34,7 @@ refer to <<xpack-security-authorization,Granting access to {kib}>>.
 . Open the main menu, and then click *Dashboard*.
 . Click **Create dashboard**.
 . Set the time range to *Last 7 days*.
-. Click **Create panel**.
-. Click **Maps**.
+. Click the **Create new Maps** icon.
```

Comment:
Mind taking a screen shot shot of icon and using the icon in the docs. Here is an example on line 8 https://github.com/elastic/kibana/blob/master/docs/maps/search.asciidoc